vkubelet/vkubelet.go

```diff
@@ -425,6 +429,73 @@ func addPodAttributes(span *trace.Span, pod *corev1.Pod) {
 	)
 }
 
+func (s *Server) onAddPod(ctx context.Context, obj interface{}) {
+	pod := obj.(*corev1.Pod)
```

Comment:
Getting a tracing span here would be great so we can measure latency on adding sending to the queue.